netsim/extra/repeater/plugin.py

```diff
@@ -0,0 +1,83 @@
+from box import Box
+from netsim import data
+from netsim.utils import log,strings
+from netsim.augment import links
+
+def clone_interfaces(link_data: Box, nodename: str, clones: list[str]) -> list[Box]:
+  """
+  update_ifindex - update any custom ifindex on the given interface
+  """
+  def update_ifindex(intf:Box, c:int) -> Box:
+    ifindex = intf.get('ifindex',None)
+    if ifindex is not None:
+      intf = data.get_box(intf)                                             # Need to make a fresh copy
+      intf.ifindex = ifindex + c + 1
+    return intf
+  
+  ifs = []
+  if nodename in [ i.node for i in link_data.interfaces ]:
+     link_data.pop('_linkname',None)
```

Comment:
I would leave the link name, that would help the user figure out where the link came from if the link generates errors.

Reply:
ok